docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -1280,6 +1287,12 @@ The output of the query has the following columns:
 * - `equality_ids`
   - `array(INTEGER)`
   - The set of field IDs used for equality comparison in equality delete files.
+* - `sort_order_id`
+  - `INTEGER`
+  - ID representing sort order for this file.
+* - `readable_metrics`
+  - `JSON`
+  - File metrics in human-readable form.
```

Comment:
It contains column size, value count, null & nan value counts, lower & upper bound. 
It can be sensitive information, but the table already has id-based lower & upper bound columns.